### PR TITLE
Added CODE_OF_CONDUCT.md

### PR DESCRIPTION
**Pull request by jebarpg**
Created the CODE_OF_CONDUCT and added [anton.osika@gmail.com](mailto:anton.osika@gmail.com) as the point of contact for any issues.

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,132 @@
+# Contributor Covenant Code of Conduct
+
+## Our Pledge
+
+We as members, contributors, and leaders pledge to make participation in our
+community a harassment-free experience for everyone, regardless of age, body
+size, visible or invisible disability, ethnicity, sex characteristics, gender
+identity and expression, level of experience, education, socio-economic status,
```
Comment (aroramrinaal) [inline CODE_OF_CONDUCT.md:8]:
```suggestion
identity or expression, level of experience, education, socio-economic status,
```

File: CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported to the community leaders responsible for enforcement at
+anton.osika@gmail.com.
+All complaints will be reviewed and investigated promptly and fairly.
+
+All community leaders are obligated to respect the privacy and security of the
+reporter of any incident.
```
Comment (aroramrinaal) [inline CODE_OF_CONDUCT.md:67]:
```suggestion
All community leaders are obligated to respect the privacy and security of reporters of incidents.
```

File: CODE_OF_CONDUCT.md [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+comments, commits, code, wiki edits, issues, and other contributions that are
+not aligned to this Code of Conduct, and will communicate reasons for moderation
+decisions when appropriate.
+
+## Scope
+
+This Code of Conduct applies within all community spaces, and also applies when
+an individual is officially representing the community in public spaces.
+Examples of representing our community include using an official e-mail address,
+posting via an official social media account, or acting as an appointed
```
Comment (aroramrinaal) [inline CODE_OF_CONDUCT.md:56]:
```suggestion
posting using an official social media account, or acting as an appointed
```

**Comment by patillacode** [comment]
Hi @jebarpg 

I believe this file should go under `.github/` folder right?
`.github\CODE_OF_CONDUCT.md`

if you pull the latest `main` branch into yours you will see the folder is already there!
Move it and I will merge, thanks!

PS: I added the minor tweaks @aroramrinaal suggested, I hope that's fine.

**Comment by patillacode** [comment]
Hi @jebarpg 

what happened? 😅

**Comment by jebarpg** [comment]
> Hi @jebarpg
> 
> what happened? 😅

I closed the old PR and created a new one.

**Pull request closed without merging**